src/Compilers/Test/Resources/Core/NetFX/Minimal/mincorlib.cs

```diff
@@ -26,6 +26,7 @@ public class Type {}
     public class ValueType {}
     public class Enum {}
     public struct Void { }
+    public struct Decimal { } // required for Mono's mcs to build a corlib
```

Comment:
It is still very much a thing. 😄 Roslyn currently only replaces NRefactory in Xamarin Studio for completion and similar services. Our entire build toolchains still use mcs and not csc.